pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -229,6 +229,9 @@ enum ServerError {
                          // use this error to indicate that this producer is now permanently
                          // fenced. Applications are now supposed to close it and create a
                          // new producer
+    TransactionComponentLoadFailed = 26; // When TransactionBuffer, TransactionPendingAck
```

Comment:
If you want to add a new error type to the PulsarApi.proto, it should start with a proposal to clarify the motivation and compatibility.